models/bill.sql

```diff
@@ -1,12 +1,21 @@
 MODEL (
     name staged.bill,
     kind FULL,
-    start '2024-04-24',
+    start '2025-01-01',
     cron '0 5 * * *',
     interval_unit 'day',
     grain (id),
     audits (
-    assert_bills_have_sponsor,
+    assert_bills_have_sponsors,
```

Comment:
Just from a formatting/legibility perspective, seems to me that the audit names should be indented